.github/scripts/set_test_package_matrix.sh

```diff
@@ -9,9 +9,11 @@ if ((RUNNER_COUNT < 1 ))
 then
   echo ERROR: RUNNER_COUNT must be greater than zero.
   exit 1 # terminate and indicate error
+else
+  EFFECTIVE_RUNNER_COUNT=$((RUNNER_COUNT-1))
```

Comment:
If `RUNNER_COUNT` is 1, we'd need to default this to 1 I think, right? `EFFECTIVE_RUNNER_COUNT` would otherwise be 0

Reply:
great call.  i've added a check for anything less than 2 and advised to use the non-matrix test in c14bf5a1aa5fe4d0fddc332b327fb634a9afd181.  otherwise we'd default to 1 and then add another array to the array of arrays and we'd spawn 2 runners when you specified 1 runner.